webrtc/RTCPeerConnection-addTransceiver.html

```diff
@@ -3,58 +3,119 @@
 <title>RTCPeerConnection.prototype.addTransceiver</title>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
+<script src="RTCPeerConnection-helper.js"></script>
 <script>
   'use strict';
 
   // Test is based on the following editor draft:
-  // https://w3c.github.io/webrtc-pc/archives/20170515/webrtc.html
+  // https://rawgit.com/w3c/webrtc-pc/cc8d80f455b86c8041d63bceb8b457f45c72aa89/webrtc.html
```

Comment:
I think that as soon as a change is merged into the Git repo, tests can be merged. It's a repeated source of confusion and frustration that the published version doesn't match, see https://github.com/w3c/webrtc-pc/issues/942

Using the rawgit URL here seems fine since even after being merged, there's no way to refer to a specific revision.